### PR TITLE
[LETS-685] add dump function for RVBT_LOG_GLOBAL_UNIQUE_STATS_COMMIT log record

### PR DESCRIPTION
**Pull request by cristiarg**
http://jira.cubrid.org/browse/LETS-685

### CODE REVIEW

File: src/storage/btree.c [hunk 1]
```diff
@@ -22459,6 +22459,38 @@ btree_rv_data_pack_btid_and_stats (const BTID * btid, long long nulls, long long
...
+
+  OR_GET_BIGINT (data_ptr, &stats.num_oids);
+  data_ptr += OR_BIGINT_SIZE;
+
+  OR_GET_BIGINT (data_ptr, &stats.num_keys);
+  data_ptr += OR_BIGINT_SIZE;
+
+  assert (CAST_BUFLEN (data_ptr - (const char *)data) == length);
+
+  fprintf (fp, "OID = page_id=%d|vol_id=%d|file_id=%d   NEW_STATS = %lld keys, %lld objects, %lld nulls:  %d\n",
```
Comment (hornetmj) [inline src/storage/btree.c]: How about replace OID = page_id=%d|vol_id=%d|file_id=%d with BTID = (%d, %d|%d) or BTREE_ID_MSG macro?

Nit:
And when I checked another debug messages, they use "nulls=%lld, oids=%lld, keys=%lld" style.

Comment (hornetmj) [reply src/storage/btree.c]: And the last %d is not used.

**Review by hornetmj — APPROVED** [review]

**Comment by hornetmj** [comment]
@cristiarg Please check the code-style violation.

File: src/storage/btree.c [hunk 2]
```diff
@@ -22459,6 +22459,38 @@ btree_rv_data_pack_btid_and_stats (const BTID * btid, long long nulls, long long
...
+  OR_GET_BIGINT (data_ptr, &stats.num_nulls);
+  data_ptr += OR_BIGINT_SIZE;
+
+  OR_GET_BIGINT (data_ptr, &stats.num_oids);
+  data_ptr += OR_BIGINT_SIZE;
+
+  OR_GET_BIGINT (data_ptr, &stats.num_keys);
+  data_ptr += OR_BIGINT_SIZE;
+
+  assert (CAST_BUFLEN (data_ptr - (const char *)data) == length);
```
Comment (wrlawodms) [inline src/storage/btree.c]: Why don't you use `btree_rv_data_unpack_btid_and_stats`?

Comment (wrlawodms) [reply src/storage/btree.c]: oh.. we don't have LOG_RCV in dummy functions... OK.  But isn't it better to create a LOG_RCV arbitrarily to make use of the existing function?

**Review by wrlawodms — APPROVED** [review]
approved with the correction from @hornetmj.

File: src/storage/btree.c (hunk 2 above)
Comment (cristiarg) [reply src/storage/btree.c]: Thanks, I will do this.
When I get back to this issue. :)

File: src/storage/btree.c (hunk 1 above)
Comment (cristiarg) [reply src/storage/btree.c]: Fixed

**Pull request merged**